hazelcast/src/main/java/com/hazelcast/sql/SqlService.java

```diff
@@ -49,4 +50,22 @@ default SqlCursor query(String sql, Object... params) {
      * @return Cursor.
      */
     SqlCursor query(SqlQuery query);
+
+    /**
+     * Execute update.
+     *
+     * @param sql SQL.
+     */
+    default void update(String sql) {
```

Comment:
I am not very keen on having such methods on public API because it increases complexity for users and we need a common method that handles all query types anyway, because otherwise, we will not be able to integrate JDBC. 
Is there a reason why the existing method or at least re-use of `SqlQuery` is not applicable for an update operation?

Reply:
If we reused existing method `SqlCursor query(SqlQuery query)` we would need to return `SqlCursor` from an execution of DDL statement, of course I could return a cursor containing single value of `0` but it just felt cumbersome - that's the thing we were discussing yesterday - I can squeeze DDL into existing method for now if we don't want to make any decision now and postpone the API decision till later.